_sources/Wavefont.designspace

```diff
@@ -3,13 +3,17 @@
     <axes>
         <axis default="100" minimum="0" maximum="100" name="Roundness" tag="ROND"/>
         <axis default="-100" minimum="-100" maximum="100" name="Vertical Element Alignment" tag="YELA">
-            <map input="-100" output="0"/>
-            <map input="100" output="1"/>
         </axis>
         <axis default="100" minimum="100" maximum="900" name="Weight" tag="wght">
             <map input="100" output="{{axes.weight.min}}"/>
             <!-- We add non-linearity to mapping -->
+            <map input="200" output="16"/>
+            <map input="300" output="33"/>
             <map input="400" output="54"/>
+            <map input="500" output="75"/>
+            <map input="600" output="100"/>
+            <map input="700" output="128"/>
+            <map input="800" output="160"/>
```

Comment:
These points have nice values and also create smooth interpolation curve: 
<img width="526" alt="image" src="https://github.com/dy/wavefont/assets/300067/b69c5175-c2bf-4deb-91ef-1ac911ffd356">
I hope it doesn't increase complexity of font calculation, since it's supposed to be performant font.
Also this needs to be aligned with linefont, because weights are on par now. (I can handle it)